cms/swap-rotating-banner.json

```diff
@@ -1,20 +1,6 @@
 {
   "$schema": "../schemas/swap-rotating-banner.schema.json",
   "banners": [
-    {
-      "name": "milkTIA",
-      "startDate": "2023-12-18T14:30:00.123Z",
-      "endDate": "2024-01-25T14:30:00.123Z",
```

Comment:
Possibly we can make use of the endDate to hide this--in case might want to re-add this later. Or perhaps you prefer that it be removed completely when we aren't using it

Reply:
I cannot edit this PR, so if you aren't going to edit--I can make my own PR

Reply:
So MilkTIA ad is already hidden, so endDate does work, and for Margined I've set it to end yesterday, so it should hide.
I've made my own PR: https://github.com/osmosis-labs/fe-content/pull/66